meta-mbl-distro/classes/mbl-distro-image-production.bbclass

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+###############################################################################
+# IMAGE_INSTALL:
+#   Specify the packages installed in the distribution images prior to
+#   inheriting from core-image to override the default behaviour.
+#
+#     packagegroup-mbl-production   production packages added for this image.
+#
+###############################################################################
+
+inherit mbl-distro-image-common
+
+IMAGE_INSTALL += "\
+    packagegroup-mbl-production \
```

Comment:
What about CORE_IMAGE_EXTRA_INSTALL - if not, should it be renamed to DEV_IMAGE_EXTRA_INSTALL?

Reply:
My first thought about not including `CORE_IMAGE_EXTRA_INSTALL` was to try to have full control of what is going to be installed in the production image and provide to the user a specific variable like `PRODUCTION_IMAGE_EXTRA_INSTALL` to include additional packages. And because this is the very early stage of the production images I'm not too worried about how to add extra packages at the moment. 

For the development image, I think we could still use `CORE_IMAGE_EXTRA_INSTALL` because this is one of the standard methods to add extra packages to an image.

It should be nice to know @jonathanhaigh-arm thoughts.

Reply:
After discussing with @jonathanhaigh-arm, we decided to move `CORE_IMAGE_EXTRA_INSTALL`  to mbl-distro-image-common class (413b547).